tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
you should not rely on the time in this very tight way. At best `check_screen` should only be used with a timeout of "0". Better use assert_screen with multiple alternatives, e.g. as in tests/x11/kontact.pm